packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -142,7 +144,7 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
     public readonly routerPath: string,
     private readonly log: Logger,
     private readonly enhanceWithContext: ContextEnhancer<any, any, any, any, any>,
-    private readonly options: RouterOptions = { isDev: false }
+    private readonly options: RouterOptions = { isDev: false, isServerless: false }
```

Comment:
We could read the default from a pseudo-config file somewhere. Defaults set within a package are sometimes hard to track down.